docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -291,13 +291,20 @@ means the document came from the local cluster.
 === Skip unavailable clusters
 
 By default, a {ccs} returns an error if *any* cluster in the request is
-unavailable.
+unavailable. You can override this default for a specific remote cluster
+using the <<skip-unavailable,`skip_unavailable`>> cluster setting.
 
-To skip an unavailable cluster during a {ccs}, set the
-<<skip-unavailable,`skip_unavailable`>> cluster setting to `true`.
+If `skip_unavailable` is `true`, a {ccs}:
 
-The following <<cluster-update-settings,cluster update settings>> API request
-changes `cluster_two`'s `skip_unavailable` setting to `true`.
+* Skips the remote cluster if it's unavailable during the search. The response's
+`_cluster.skipped` value contains a count of any skipped clusters.
+
+* Ignores the <<search-partial-responses,`allow_partial_search_results`>>
+parameter, if specified, when searching the remote cluster. This means searches
+on the remote cluster may return partial results.
```

Comment:
In my testing, `ignore_unavailable` still worked, regardless of the `skip_unavailable` setting. I left it out of my revision, but feel free to re-add if I'm missing something.

Reply:
that should not really be the case, I mean you can still ignore unavailable indices, but when you set it to `false`, `skip_unavailable` will prevent you from getting back the expected errors.